atmo/clusters/jobs.py

```diff
@@ -72,9 +72,11 @@ def update_clusters_info():
     """
     # only update the cluster info for clusters that are pending
     active_clusters = Cluster.objects.active()
+    # get the datetime of the oldest active cluster, set to the start of the day
```

Comment:
The start_date in our database is not the same as the date that AWS filters cluster by, we don't sync it. To prevent race conditions in which our start date is a bit earlier due to latency via the API call I think it's safest to just check the days servers.